### ISSUE TITLE
listVideoInputDevices is giving 'AVStream Media Device' in the list of available videos

### ISSUE THREAD

**Issue by usharanitalatala** [zxing-js/library#375]
Hi Team,

We are getting 'AVStream Media Device' in the list of available videos. Could you please let me know if there are any other method that give distinction on Integrated Webcam vs Video Drivers?

**Comment by odahcam** [comment]
Nope, thereisn't but we do filter the video input devices, as the method name suggests. You could make use of the WebRTC APIs  yourself to catch and filter the devices you want and pass it to the library.
